docs/content/v0.20/packaging.md

```diff
@@ -12,19 +20,24 @@ There is also a container image involved.
 
 ### Derived files
 
-Some files in the kubestellar repo are derived from other files there.
+Some files in the kubestellar repo are derived from other files there. These are all generated by contributors and kept in GitHub. A contributor can use the individual `make` commands mentioned below to update subsets, or use the following command to update them all.
 
-#### Files generated by controller-gen
+```
+make all-generated
+```
 
-- `make manifests` generates the CustomeResourceDefinition files.
+## Files generated by controller-gen
 
-- `make generated` generates the deep copy code.
+- `make manifests` generates the CustomeResourceDefinition files,
+  which exist in two places: [config/crd/bases](config/crd/bases) and
+  [pkg/crd/files](pkg/crd/files).
 
-Both of the above are currently broken, see Issue 1611.
+- `make generated` generates the deep copy code, which exists in
+  `zz_generated.deepcopy.go` next to the API source.
```

Comment:
what about `make codegenclients` and `make all-generated`?

Reply:
I am not sure I understand the question. The text already says that `make all-generated` is equivalent to the combination of the three other make targets for generating files. The text already says that `make codegenclients` makes the files in `pkg/generated/`, not files in `api/`.

Reply:
oops. I missed it in the doc. saw the make all-generated only in makefile.
then all good.